src/Apps/ViewingRoom/Routes/Works/Components/ViewingRoomArtworkDetails.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import { Box, Sans, Button, Serif } from "@artsy/palette"
+
+export const ViewingRoomArtworkDetails = props => {
+  return (
+    <Box width="50%" m="auto">
+      <Box>
+        <Sans size="3">Christine Sun Kim</Sans>
```

Comment:
Do we do like `<Sans as="h2">` or anything like that in places?

Reply:
Ah, nice call -- we can't do `as` for "reasons" but we have an "element" prop we can use instead.